### PR TITLE
Upgrade node

### LINKED ISSUE #132: Support for NodeJS10.X run environment
With the NodeJS8 rim environment coming to end of life we need to support nodejs10. 

Current Issues: 
 
- Nodejs10.X runtime is VERY lightweight compared to the node8 one, this has broken multiple pieces.

### PR DESCRIPTION
**Pull request by mhaber-tophat**
fixes #132 

Upgrades Node8 > Node10. This upgrade had a couple of caveats that are addressed here in order for the upgrade to work.

1. adds a find.zip that packages up the bash find command (compiled for amazonlinux) and adds it as a layer to the lambda. This is needed because the amazonlinux2 OS that the nodejs10.x runtime uses is very lightweight and removed this command. JEST has a dependency on the `find` command existing and being the in the path. If jest ever addresses this issue we can remove this added layer. related issue
: https://github.com/facebook/jest/issues/4211

2. the chrome image we were using broke on nodejs10.x. This is due to missing library files + font files taken away in amazonlinux2. (https://github.com/adieuadieu/serverless-chrome/issues/203). To fix this we are moving away from relying on serverless-chrome and moving over to using chrome-aws-lamba (https://github.com/alixaxel/chrome-aws-lambda) as were we obtain chrome from. With the latest versions of this project, it comes with a compiled chrome ready for the nodejs10.x runtime without the need of layers. 🎆   With the usage of this project we were able to rip out all our previous logic of having to load chrome + store in s3. The logic is ALOT simpler now as the chrome-aws-lambda does most of it for us :) 

3. updated the source of the waitpid2 package we are using as the previous one wasnt compiled/compatable with node10. 

have tested internally with our sanity test suite. if you want to test, just pull thm-sanity + set `  "lambdaFunction": "sanity-runner-upgrade-node-asiaqefsjrmxs6aizlkj"` in `config.json`

### CODE REVIEW

**Comment by eastenluis** [comment]
Does this mean all users will need newer node version to run the tests?

**Comment by mhaber-tophat** [comment]
> Does this mean all users will need newer node version to run the tests?

no. this is just on the backend. i was testing with packaging tests with node8 + webpack and its all fine. This is cause node8 runtime is getting deprecated and in feb we wont be able to publish new versions of the lambda if we didnt do this.

File: service/package.json [hunk 1]
```diff
@@ -36,9 +38,10 @@
     "jest-junit": "^6.3.0",
     "node-pre-gyp": "^0.12.0",
     "posix": "^4.1.2",
-    "puppeteer": "1.11.0",
+    "puppeteer-core": "^2.0.0",
     "tar": "^4.4.4",
     "uuid": "^3.2.1",
-    "waitpid2": "git://github.com/joshiggins/node-waitpid2.git#2270e88"
+    "waitpid2": "git://github.com/unbounce/node-waitpid2"
+
```
Comment (chrono) [inline service/package.json]: was this a manual edit? (why this extra line?)

Comment (chrono) [reply service/package.json]: i guess package.json is fine to edit by hand, just would prefer to keep a canonical json representation

**Pull request merged**